canard.h

```diff
@@ -118,8 +124,15 @@ extern "C" {
 #define CANARD_MAX_TRANSFER_PAYLOAD_LEN             ((1U << CANARD_TRANSFER_PAYLOAD_LEN_BITS) - 1U)
 
 
+typedef enum
+{
+    CanardTransportProtocolCAN,
```

Comment:
```suggestion
    CanardTransportProtocolCAN2,
```

`CAN2` is short for "CAN 2.0".

Reply:
great! willfix